packages/core/stories/card.doc.stories.mdx

```diff
@@ -38,6 +38,8 @@ Cards can have interactable styling by setting `interactable={true}`. These card
 
 Cards can have disabled styling applied by setting `disabled={true}`.
 
+Make sure to as well apply `disabled={true}` to any nested components, such as Text, to ensure disabled styling is also applied to the inner content. We discourage the use of links within interactable Cards that can be disabled, since Salt Links cannot be disabled out of the box as they present usability issues, and create misleading interactions within an application.
```

Comment:
I think we discourage the use of any focusable elements inside the interactable cards regardless of disabled state. Or are we not?

Reply:
yes - this is just a bit design point because of the fact we can disable Text, people may not realise we can't disable Link. I will add to the docs about that too

Reply:
I actually find it quite confusing reading this.

Reply:
I would agree with @alycrys the confusion here is that we contradict ourselves by earlier saying focusable elements shouldn't be inside interactable cards and then later say links should not be inside interactable cards.

Regardless of the fact whether links could be disabled or not it is still focusable/interactable therefore shouldn't be inside the card anyway

Reply:
sure, i copied it from figma. will delete this bit